cmake/ConkyPlatformChecks.cmake

```diff
@@ -249,14 +253,39 @@ endif(BUILD_HTTP)
 
 if(BUILD_NCURSES)
   set(CURSES_NEED_NCURSES TRUE)
-  include(FindCurses)
 
-  if(NOT CURSES_FOUND)
-    message(FATAL_ERROR "Unable to find ncurses library")
-  endif(NOT CURSES_FOUND)
+  find_path(NCURSES_INCLUDE_PATH
+    NAMES ncurses.h curses.h
```

Comment:
Check only for `curses.h`, we want to be evil and fail here if `ncurses.h` exists and `curses.h` re-export doesn't (it shouldn't; checked on Arch, NetBDS diff also replaces it with `curses.h`).